crd-docs/crd/kiali.io_kialis.yaml

```
@@ -1246,6 +1249,10 @@ spec:
                         description: "The name of the project where the Dashboards are defined."
                         type: string
                         default: "istio"
+                      url_format:
+                        description: "The url format. Use `openshift` when used with Perses Dashboards from the cluster observability operator in OpenShift. Use `default` for standard Perses upstream. "
+                        type: string
+                        enum: ["", "default", "openshift"]
```

```suggestion
                      url_format:
                        description: "The url format. Use `openshift` when using Perses Dashboards via the Cluster Observability operator in OpenShift. Use `default` for standard Perses upstream. "
                        type: string
                        default: "default"
                        enum: ["default", "openshift"]
```

This was a little confusing to me. I'm unsure when I should use `""`, and I don't really like an enum value of `""`. I suggest we just set the default to "default". Meaning, you only have to set it for OpenShift.